api/src/main/java/ru/urururu/sanity/api/cfg/CfePrinter.java

```diff
@@ -8,62 +8,21 @@
 /**
  * @author <a href="mailto:dmitriy.g.matveev@gmail.com">Dmitry Matveev</a>
  */
-public class CfePrinter implements CfeVisitor {
-    final StringBuilder sb = new StringBuilder();
-    final Map<RValue, Integer> tmpVars = new HashMap<>();
-    final Map<Cfe, Integer> cfeIds = new HashMap<>();
-    final Set<Cfe> printed = new HashSet<>();
-
-    @Override
-    public String toString() {
-        return sb.toString();
-    }
-
-    private boolean isEmpty() {
-        return sb.length() == 0;
-    }
+public class CfePrinter {
+    public static CfePrinter DEFAULT = new CfePrinter();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "DEFAULT" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3008)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "public static DEFAULT" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1444)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make DEFAULT a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)